main.typ

```diff
@@ -40,6 +40,20 @@
 
 = Typical Open Source Development
 
+Open source software development is a collaborative effort between developers and users. The users of the software are able to contribute to the development of the software by reporting bugs, suggesting features, and even contributing code. The developers of the software are able to use the feedback from the users to improve the software. Open source software development differs from traditional software development in two major ways. The first being that the users are able to contribute to the development of the software. The second being that almost all collaboration between developers happens asynchronously, through the internet.
+
+Often times, contributors to open source software development are completely anonymous. This is because the contributors are not required to provide any personal information when contributing to the software. This is a major difference from traditional software development, where the developers are usually working in the same location and are able to communicate face to face. It creates a completely different dynamic when managing a project.
```

Comment:
- Generally instead of "often times" ==> too informal
- Cut "This is because" unnecessary and informal
- Compared to the face-to-face interaction of developers in a traditional software development environment, the open-source development environment differs in that developers work asynchronously and anonymously.   
- The dynamic when managing a project are stark opposites comparing online collaboration versus in-person synergy.